src/chapters/more-on-functions/why-use-anonymous-functions.rst

```diff
@@ -0,0 +1,38 @@
+Why Use Anonymous Functions?
+============================
+
+At this point, you may be asking yourself why you are learning anonymous functions. They seem strange, and their utility may not be immediately obvious. While the opinions of programmers differ, we will discuss the two main reasons that *we* think anonymous functions are important to understand.
+
+Anonymous Functions Can Be Single-Use
+-------------------------------------
+
+There are many situations in which you will need to create a function that will only be used once. To see this, recall one of our earlier examples.
+
+.. admonition:: Example
+
+   The anonymous function created in this example cannot be used outside of ``setTimeout``.
+
+   .. sourcecode:: js
+   
+      setTimeout(function () {
+         console.log("The future is now!");
+      }, 5000);
+
+Defining an anonymous function at the same time it is passed as an argument prevents it from being used elsewhere in the program.
+
+Additionally, in programs that use lots of functions---such as web applications, as you will soon learn---defining functions anonymously, and directy within a function call, can reduce the number of names you need to come up with. 
+
+Anonymous Functions Are Ubiquitous in JavaScript
+------------------------------------------------
+
+JavaScript programmers use anonymous functions *a lot*. Many programmers use anonymous functions with the same gusto as that friend of yours that puts hot sauce on *everything*. For these programmers, just because an anonymous function isn't need to solve a problem doesn't mean that it *shouldn't* be used to solve the problem. Avoiding code that uses anonymous functions is impossible, since they are used so widely by so many JavaScript developers.
```

Comment:
Grammar: "...that friend of yours who puts hot sauce on *everything*."

New paragraph with typo fix: "Just because an anonymous function isn't needed to..."

Suggestion: "...since they appear EVERYWHERE."  (Hyperbole appropriate?)

Reply:
The hyperbole feels a bit strong. I shortened this to, "Avoiding JavaScript code that uses anonymous functions is impossible." Better?

Reply:
Yep.  Better.